timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/graphql/mutations/ResourceSyncUpdateMutation.java

```diff
@@ -48,7 +49,7 @@ public Object get(DataFetchingEnvironment env) {
         resourceSyncFileLoader, dataSet.get(), false);
       String capabilityListUri = dataSet.get().getMetadata().getImportSource();
       resourceSyncReport = resourceSyncImport.filterAndImport(capabilityListUri, null, true,
-        null);
+        authString);
```

Comment:
For the resourcesync we want the authorization token for the remote resourcesync.